docs/aspnetcore-facility.md

```diff
@@ -51,7 +91,7 @@ Here is a complete example:
 ```csharp
 public class Startup
 {
-	private readonly WindsorContainer container = new WindsorContainer();
+	public static readonly WindsorContainer container = new WindsorContainer();
```

Comment:
Is having this public in the example a good idea?

Reply:
This should be private. Agreed.

Reply:
Done